hbase-handler/pom.xml

```diff
@@ -186,6 +186,7 @@
       <artifactId>jetty-runner</artifactId>
       <version>${jetty.version}</version>
       <scope>test</scope>
+      <classifier>shaded</classifier>
```

Comment:
what is the reason for this line?

Reply:
Without the classifier, the dependency is looking for a jar named `jetty-runner-9.4.51.v20230217.jar`.

With the classifier, the dependency is looking for a shaded jar named `jetty-runner-9.4.51.v20230217-shaded.jar`.

The new project creates a jar locally which is named `jetty-runner-shaded-1.0-SNAPSHOT-shaded.jar` and then we install it under the local `.m2/repository` with the name `jetty-runner-9.4.51.v20230217-shaded.jar`.

The installation happens with this command

```
mvn install:install-file -Dfile=target/jetty-runner-shaded-1.0-SNAPSHOT-shaded.jar -DgroupId=org.eclipse.jetty -DartifactId=jetty-runner -Dversion=9.4.51.v20230217 -Dclassifier=shaded -Dpackaging=jar
```

Reply:
I've added a README explaining all this here

https://github.com/xBis7/docker-setup-helper-scripts/blob/jetty-runner-shaded-project/jetty-runner-shaded/README.md

Reply:
ok, thanks.